source/extensions/filters/http/jwt_authn/verifier.cc

```diff
@@ -258,39 +258,30 @@ class AnyVerifierImpl : public BaseGroupVerifierImpl {
                   const Protobuf::Map<std::string, JwtProvider>& providers,
                   const BaseVerifierImpl* parent)
       : BaseGroupVerifierImpl(parent) {
-    const JwtRequirement* by_pass_type_requirement = nullptr;
-    std::vector<std::string> used_providers;
+
     for (const auto& it : or_list.requirements()) {
-      bool is_regular_requirement = true;
       switch (it.requires_type_case()) {
-      case JwtRequirement::RequiresTypeCase::kProviderName:
-        used_providers.emplace_back(it.provider_name());
-        break;
-      case JwtRequirement::RequiresTypeCase::kProviderAndAudiences:
-        used_providers.emplace_back(it.provider_and_audiences().provider_name());
-        break;
       case JwtRequirement::RequiresTypeCase::kAllowMissingOrFailed:
+        is_allow_failed_ = true;
```

Comment:
should this be called `is_allow_missing_or_failed_`?

Reply:
Done